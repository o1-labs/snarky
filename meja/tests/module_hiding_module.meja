module A = {
  module X = {};
};

module B = {
  module A = {
    module Y = {};
  };
};

open B;

module Y = A.Y;

/* Expect an error, A should be out of scope. */
module X = A.X;
