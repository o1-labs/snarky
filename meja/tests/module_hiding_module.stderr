File "tests/module_hiding_module.meja", line 16, characters 11-14:
Error: Unbound module A.X.
